LocalMultiplayerAgent/DeploymentTool/deployment.json

```diff
@@ -0,0 +1,16 @@
+{
+    "BuildName": "GameContainerW",
+    "VmSize": "Standard_D2_v2",
+    "MultiplayerServerCountPerVm": 10,
+    "RegionConfigurations": [
+        {
+            "Region": "EastUs",
+            "MaxServers": 10,
+            "StandbyServers": 5
+        }
+    ],
+    "OSPlatform": "Windows",
+    "AssetFileNames": [
```

Comment:
Why can't we use the AssetDetails from MultiplayerSettings.json?

Reply:
Yes I was supposed to remove this one. I will do that in my next commit